Source/WTF/wtf/text/StringConcatenate.h

```diff
@@ -185,6 +185,9 @@ template<> class StringTypeAdapter<StringImpl*, void> {
     bool is8Bit() const { return !m_string || m_string->is8Bit(); }
     template<typename CharacterType> void writeTo(CharacterType* destination) const
     {
+        if (!m_string)
+            return;
```

Comment:
This is closer to the right place, but why is the code below crashing with `m_string` of null? Constructing a `StringView` from a null string should work. And `getCharactersWithUpconvert` should work too. Where is the crash occurring?

Reply:
The crash backtrace is  in https://bugs.webkit.org/show_bug.cgi?id=245640